test/lit/passes/simplify-globals-gc.wast

```diff
@@ -31,3 +31,27 @@
  )
 )
 
+(module
+ ;; CHECK:      (type $struct (struct ))
+ (type $struct (struct ))
+
+ ;; CHECK:      (type $1 (func (result anyref)))
+
+ ;; CHECK:      (global $a (ref $struct) (struct.new_default $struct))
+ (global $a (ref $struct) (struct.new_default $struct))
+ ;; CHECK:      (global $b (ref $struct) (global.get $a))
+ (global $b (ref $struct) (global.get $a))
```

Comment:
What is the role of `$b` in this test?

Reply:
Just to be comprehensive. `$c` is equal to both `$a` and `$b` but in different ways (it reads `$a`, but it has equal content to `$b`). I don't think we optimize both cases atm but this would handle things if we ever did.